x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -141,66 +141,68 @@ export const useCasesColumns = ({
     },
   ];
 
-  if (caseAssignmentAuthorized) {
+  if (caseAssignmentAuthorized && !isSelectorView) {
```

Comment:
All these if blocks are starting to make this hook a little complicated. You don't need to do the refactoring in this PR but how about we create an issue to track splitting the hook into two, one for the modal and one for the cases table and they share the individual column functionality but don't need all the if blocks.

Reply:
yup I agree, it has become a mess right now. It is better to separate in two hooks.